sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/UnnecessaryImportsCheck.kt

```diff
@@ -187,9 +251,14 @@ private fun KtImportDirective.isImportedImplicitlyAlready(containingPackage: Str
 
 private fun KtReferenceExpression.importableSimpleName() =
     when (this) {
-        is KtOperationReferenceExpression -> operationSignTokenType
-            ?.let { (OperatorConventions.BINARY_OPERATION_NAMES[it] ?: OperatorConventions.UNARY_OPERATION_NAMES[it])?.asString() }
-            ?: getReferencedName()
+        is KtOperationReferenceExpression ->
+            operationSignTokenType?.let { token ->
+                (deparenthesize().parent as? KtPrefixExpression)
+                    ?.let { OperatorConventions.UNARY_OPERATION_NAMES[token] }
+                    ?: (OperatorConventions.getNameForOperationSymbol(token))
```

Comment:
I had a hard time understanding why there is a `?.let` here without using `it` that this is really an if-else block. I suggest using an explicit if-else here:
```suggestion
                if (deparenthesize().parent is KtPrefixExpression) OperatorConventions.UNARY_OPERATION_NAMES[token]
                else OperatorConventions.getNameForOperationSymbol(token)
```